Sample Data/ASIM/VMware_CarbonBlackCloud_ASimNetworkSessionThreat_IngestedLogs.csv

```diff
@@ -0,0 +1,9 @@
"reason_code_s","backend_timestamp_t [UTC]","report_link_s","alert_notes_present_b","parent_cmdline_s","parent_guid_s","device_target_value_s","ioc_hit_s","device_external_ip_s","device_policy_id_d","device_os_version_s","policy_applied_s","parent_effective_reputation_s","process_name_s","version_s","device_location_s","report_description_s","threat_id_g","is_updated_b","parent_username_s","device_name_s","org_key_s","watchlists_s","workflow_status_s","workflow_change_timestamp_t [UTC]","workflow_changed_by_type_s","workflow_changed_by_s","workflow_closure_reason_s","process_effective_reputation_s","parent_name_s","process_publisher_s","mdr_alert_notes_present_b","process_md5_g","device_id_d","ml_classification_org_prevalence_s","sensor_action_s","device_username_s","backend_update_timestamp_t [UTC]","last_event_timestamp_t [UTC]","threatInfo_incidentId_g","threatInfo_score_d","threatInfo_summary_s","threatInfo_time_d","threatInfo_indicators_s","threatInfo_threatCause_reason_s","threatInfo_threatCause_reputation_s","threatInfo_threatCause_actor_s","threatInfo_threatCause_actorName_s","threatInfo_threatCause_actorProcessPPid_s","threatInfo_threatCause_threatCategory_s","threatInfo_threatCause_originSourceType_s","threatInfo_threatCause_causeEventId_g","threatInfo_threatCause_processGuid_s","threatInfo_threatCause_parentGuid_s","threatHunterInfo_incidentId_g","threatHunterInfo_score_d","threatHunterInfo_summary_s","threatHunterInfo_time_d","threatHunterInfo_indicators_s","threatHunterInfo_watchLists_s","threatHunterInfo_iocId_g","threatHunterInfo_count_d","threatHunterInfo_dismissed_b","threatHunterInfo_documentGuid_s","threatHunterInfo_firstActivityTime_d","threatHunterInfo_md5_g","threatHunterInfo_policyId_d","threatHunterInfo_processGuid_s","threatHunterInfo_processPath_s","threatHunterInfo_reportName_s","threatHunterInfo_reportId_s","threatHunterInfo_reputation_s","threatHunterInfo_responseAlarmId_g","threatHunterInfo_responseSeverity_d","threatHunterInfo_runState_s","threatHunterInfo_sha256_s","threatHunterInfo_targetPriority_s","threatHunterInfo_threatCause_reason_s","threatHunterInfo_threatCause_actorProcessPPid_s","threatHunterInfo_threatCause_parentGuid_s","threatHunterInfo_threatCause_causeEventId_s","threatHunterInfo_threatCause_reputation_s","threatHunterInfo_threatCause_actor_s","threatHunterInfo_threatCause_actorName_s","threatHunterInfo_threatCause_threatCategory_s","threatHunterInfo_threatCause_originSourceType_s","threatHunterInfo_threatCause_processGuid_s","threatHunterInfo_threatId_g","threatHunterInfo_lastUpdatedTime_d","threatHunterInfo_orgId_d","url_s","type_s","eventDescription_s","deviceInfo_internalIpAddress_s","deviceInfo_externalIpAddress_s","deviceInfo_targetPriorityCode_d","deviceInfo_groupName_s","deviceInfo_deviceId_d","deviceInfo_deviceName_s","deviceInfo_deviceType_s","deviceInfo_deviceVersion_s","deviceInfo_email_s","deviceInfo_targetPriorityType_s","deviceInfo_uemId_s","ruleName_s","eventTime_d",Type,"_ResourceId"
```

Comment:
The threat info joining with their Activity in sample logs are curated or is that the actual data?
Wanted to understand in general if 'Threat' info making sense after joining with schema specific logs?
Threat logs joining with actual activity would give us more confidence. Among the logs that are uploaded is there any that is an actual pair?

Reply:
Yes. The are actual pairs available in sampledata.
you can check for threatInfo_incidentId_g=2fd9b668-8ca6-1d3e-b3ce-6938867e2a43 in VMware_CarbonBlackCloud_ASimNetworkSessionThreat_RawLogs.json and same is available in VMware_CarbonBlackCloud_ASimNetworkSession_RawLogs.json as incident_id=2fd9b668-8ca6-1d3e-b3ce-6938867e2a43